lib/detect-testing-library-utils.ts

```diff
@@ -580,14 +581,25 @@ export function detectTestingLibraryUtils<
       return isRenderUtil(initIdentifierNode);
     };
 
-    const isDebugUtil: IsDebugUtilFn = (node) => {
-      return isTestingLibraryUtil(
-        node,
-        (identifierNodeName, originalNodeName) => {
-          return [identifierNodeName, originalNodeName]
-            .filter(Boolean)
-            .includes('debug');
-        }
+    const isDebugUtil: IsDebugUtilFn = (identifierNode) => {
+      const isBuiltInConsole =
+        isMemberExpression(identifierNode.parent) &&
+        identifierNode.parent.parent &&
+        isCallExpression(identifierNode.parent.parent) &&
+        isMemberExpression(identifierNode.parent.parent.callee) &&
+        ASTUtils.isIdentifier(identifierNode.parent.parent.callee.object) &&
+        identifierNode.parent.parent.callee.object.name === 'console';
```

Comment:
This is redundant, isn't it? `identifierNode.parent.parent.callee.object.name` is just `identifierNode.parent.object.name` right? Can you simplify this check please?